.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+exclude = .git, .github, __pycache__ , .vscode, build
+max-line-length = 99
+ignore = E302,E305,W503,E203,E731,E402,E266,E712,F401,F821
```

Comment:
E032 : 2 blank k-lines between funcitons --> should be kept
E305 : expected 2 blank lines after function --> should be kept
W503 : Line break occurred before a binary operator --> kept
E203 : white space before ':' --> kept
E402	Module level import not at top of file --> kept
E266	Too many leading '#' for block comment --> kept
E712	Comparison to true should be 'if cond is true:' or 'if cond:' --> kept

F401	Module imported but unused --> kept
F821	Undefined name name --> kept